src/MahApps.Metro/MahApps.Metro.Shared/Controls/Helper/TextBoxHelper.cs

```diff
@@ -424,7 +430,41 @@ public static void SetUseFloatingWatermark(DependencyObject obj, bool value)
         {
             obj.SetValue(UseFloatingWatermarkProperty, value);
         }
-        
+
+        [Category(AppName.MahApps)]
+        [AttachedPropertyBrowsableForType(typeof(TextBoxBase))]
+        [AttachedPropertyBrowsableForType(typeof(PasswordBox))]
+        [AttachedPropertyBrowsableForType(typeof(ComboBox))]
+        [AttachedPropertyBrowsableForType(typeof(NumericUpDown))]
+        [AttachedPropertyBrowsableForType(typeof(DatePicker))]
+        [AttachedPropertyBrowsableForType(typeof(TimePickerBase))]
+        public static string GetFloatingWatermarkLocation(DependencyObject obj)
```

Comment:
Why returning a string when FloatingWatermarkLocation is of type `FloatingWatermarkLocations`?